index.html

```diff
@@ -0,0 +1,168 @@
+  <link rel="icon" href="./images/cap-1266204__340.webp">
+</head>
+
+<body>
+  <header>
+    <div class="contact-container">
+      <div></div>
+      <div class="contact">
+        <ul class="contact-ul">
+          <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+          <li><a href=""><i class="fa-brands fa-twitter"></i></a></li>
+          <li class="contact-li"><a href="">English</a></li>
+          <li class="contact-li"><a href="">My Page</a></li>
+          <li class="contact-li"><a href="">sign Up</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="header-container">
+      <div class="header-container">
+        <div class="nav-container">
+          <div class="logo">
+            <div>
+              <ul>
+                <li>
+                  <i class="fa-solid fa-graduation-cap cap"></i>
+                  <a class="logo-link" href="#home.html">Educational Resources</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="nav">
+            <ul class="nav-ul">
+              <li class="navigation" id="home.html"><a href="#">Home</a></li>
+              <li class="navigation"><a href="about.html">About</a></li>
+              <li class="navigation"><a href="#event">Event</a></li>
+              <li class="navigation"><a href="#guest">Guest</a></li>
+              <li class="navigation sponsor"><a href="#sponsor">Sponsor</a></li>
+            </ul>
+          </div>
+
+          <div class="mobile-menu">
+            <ul class="menu-list">
+              <li><a class="mobile-link" href="home.html">Home</a></li>
+              <li><a class="mobile-link" href="about.html">About</a></li>
+              <li><a class="mobile-link" href="#event">Event</a></li>
+              <li><a class="mobile-link" href="#guest">Guest</a></li>
+              <li><a class="mobile-link" href="#sponsor">Sponsor</a></li>
+
+            </ul>
+          </div>
+
+          <div class="hamburger">
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </header>
+  <section class="headline">
+    <div class="headline-container">
+      <p class="headline-p">"Hello! scholar"</p>
+      <h1>Educational Resources Summit 2023</h1>
+      <p class="headline-summary">
+        Educational resources are used in a learning environment to help and assist with people’s development and
+        learning. We are designed to reinforce learning and in some cases allow people to put their knowledge to the
+        test. Educational resources are brilliant for teachers and educators to help them deliver the best quality
+        lessons.
+      </p>
+      <h2>
+        2023.02.20(THU) ~ 21(FRI)
+      </h2>
+      <P class="venue">
+        @ Educational Resources center, ZONE 7 FCT Abuja Nigeria.
+      </P>
+    </div>
+  </section>
```

Comment:
Nice work building the headline section, let's make it even better by making it match the design😉 Kindly make the intro section have a view height of 100% as it is in the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)😉😉

<img width="948" alt="your intro" src="https://user-images.githubusercontent.com/64643995/225452783-3dcd724d-8f2f-4a2a-8777-f540a1069ef9.png">